src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -114,8 +56,21 @@ public Drivetrain() {
    * @param thro
    */
   public void FlyByWireA(double steerage, double thro){
-    starboardMotor.set(ControlMode.PercentOutput, (thro + (steerage)) * -1); // Set the starboard motor to the sum of thro - steerage
-    portMotor.set(ControlMode.PercentOutput, (thro - (steerage)) * -1); // Set the port motor to the sum of thro + steerage
+    starboardMotor.set(thro + (steerage)); // Set the starboard motor to the sum of thro - steerage
+    portMotor.set(thro - (steerage)); // Set the port motor to the sum of thro + steerage
+  }
+
```

Comment:
We had removed some inversions here. They should be done solely in lines 37 and 38 of drivetrain.java

Reply:
Complicated inversions will be resolved in `driver-cleanup`